frontend/uddhar-frontend/src/coordinator/pages/sub-page/LeftPanel.jsx

```
@@ -1,23 +1,22 @@
+    { name: "Analytics", icon: icons.analysis },
+    { name: "Resources", icon: icons.resources, badge: "14 Events" },
+    { name: "Communication", icon: icons.communication },
+    { name: "Emergency", icon: icons.emergency },
   ];
 
   return (
-    <div className="w-70 h-screen bg-white shadow-lg p-4 flex flex-col my-5">
+    <div className="w-70 h-screen bg-white shadow-lg p-4 flex flex-col my-5 sticky top-0">
       <div className="relative mt-4 mb-4">
-        <img className="absolute left-3 top-2.5 text-gray-400 w-5 h-5" src={search} />
+        <img className="absolute left-3 top-2.5 text-gray-400 w-5 h-5" src={icons.search} />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)